user_docs/en/userGuide.md

```diff
@@ -2163,6 +2163,10 @@ This means that you do not have to scroll the display at the end of each line ev
 This may allow for more fluent reading of large amounts of text.
 It is disabled by default.
 
+##### Show paragraphs start {#BrailleSettingsShowParagraphStart}
+
+- If enabled and "Read by paragraph" is checked, two spaces will be displayed to indicate the start of a paragraph.
```

Comment:
* remove leading dash
* Link the "Read by paragraph" option

```suggestion
If enabled and "[Read by paragraph](#BrailleSettingsReadByParagraph)" is checked, two spaces will be displayed to indicate the start of a paragraph.
```

Moreover, it would be worth rephrasing this paragraph with a bit more details. But I am not familiar with "Read by paragraph" option and will not be able to test on a braille display before next week, so I won't be able to make a detailed suggestion.

Maybe something like the following (to be adapted and polished):
When "Read by paragraph" option is enabled, we face [issueToBeDescribed, e.g. difficulty to identify paragraph start].
Enabling this option enable to mitigate this issue adding two spaces at the beginning of paragraphs.